.github/workflows/scripts/script.sh

```diff
@@ -145,7 +145,7 @@ if [[ "$TEST" == "upgrade" ]]; then
 
   # Running post upgrade tests
   git checkout ci_upgrade_test -- pulp_container/tests/
-  pytest -vv -r sx --color=yes --pyargs --capture=no --durations=0 pulp_container.tests.upgrade.post
+  pytest -v -r sx --color=yes --pyargs -capture=no pulp_container.tests.upgrade.post
```

Comment:
Is this correct?
@fao89

Reply:
yep, I forgot to change back, I added these to debug

Reply:
And what about the double dash before "capture" ?

Reply:
I didn't notice it, I need to change it at the plugin-template

Reply:
done - https://github.com/pulp/plugin_template/pull/486